### ISSUE TITLE
[Bug]: crashed 

### ISSUE THREAD

**Issue by vAHiD55555** [AndroidIDEOfficial/AndroidIDE#1610]
### What happened?

Ca  not clone

### What's the expected behavior?

![Uploading Screenshot_20240106_195742_AndroidIDE.jpg…]()

### What version of AndroidIDE you're using?

v2.6.0 (debug builds)

### Relevant log output

```shell
AndroidIDE Crash Report
Version : v2.7.0-beta (27001)
CI Build : true
Branch : main
Commit : 062d240e9
Variant : arm64-v8a (release)
Build type : OFFICIAL
SDK Version : 34
Supported ABIs : [arm64-v8a, armeabi-v7a, armeabi]
Manufacturer : samsung
Device : SM-A536E

Stacktrace:
java.lang.RuntimeException: Can't create handler inside thread Thread[DefaultDispatcher-worker-1,5,main] that has not called Looper.prepare()
	at android.os.Handler.<init>(Handler.java:227)
	at android.os.Handler.<init>(Handler.java:129)
	at android.app.Dialog.<init>(Dialog.java:168)
	at android.app.Dialog.<init>(Dialog.java:279)
	at androidx.activity.ComponentDialog.<init>(Unknown Source:5)
	at androidx.appcompat.app.AppCompatDialog.<init>(Unknown Source:22)
	at androidx.appcompat.app.AlertDialog.<init>(Unknown Source:4)
	at org.commonmark.internal.BlockContent.create(Unknown Source:10)
	at com.google.android.material.dialog.MaterialAlertDialogBuilder.create(Unknown Source:0)
	at org.commonmark.internal.BlockContent.show(Unknown Source:0)
	at androidx.room.SharedSQLiteStatement$stmt$2.invoke(SourceFile:69)
	at com.itsaky.androidide.fragments.MainFragment$doClone$cloneJob$1.invokeSuspend(Unknown Source:135)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:8)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:120)
	at org.greenrobot.eventbus.AsyncPoster.run(Unknown Source:12)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Unknown Source:2)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(Unknown Source:91)
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by arshiasahhamte** [comment]
Hello. Happy new year to all the staff of Yasmin's friend. My friend had a previous stroke, life is hard, I can't walk much, I have heart palpitations. I started the new year with tears, friends. Now I don't want to swap because of 50 trons I don't know where to get 50 trons, it's 10 dollars, who should I tell? Now I put the Android app for Koshi, it says give 50 dollars or 20 dollars. My friends, I stayed for 10 dollars of trons. God send me 10 dollars wallet address, give me pills.
